contracts/exchange/Erc20Escrow.sol

```diff
@@ -154,13 +154,32 @@ contract Erc20Escrow is IErc20Escrow, EscrowBase {
         }
     }
 
-    // Deposit Platform Fees
+    // Transfer Creator Royalty from escrowed buy order (single id) to escrow
+    function transferRoyalty(
+        uint256 _orderId,
+        address _owner,
+        uint256 _amount
+    ) external override onlyRole(MANAGER_ROLE) {        
```

Comment:
convert this to a "public" function instead of an "external" function and have the multiple ids version of transferRoyalty() just call the single version of royaltyId like so:

essentially, replace lines 142 to 152 with:
transferRoyalty(_orderIds[i], _amounts[i], _owner);